src/app/util/util.js

```diff
@@ -42,3 +42,28 @@ NavPill.propTypes = {
   to: PropTypes.string,
   children: PropTypes.node
 };
+
+/**
+ * Dates
+ */
+import {format} from 'date-fns';
+import * as locales from 'date-fns/locale'; // we are ok with importing all languages
+/**
+ * Call's `date-fns/format` with a locale from `navigator.language`
+ *
+ * @param {Date | number | string} date - a date to format
```

Comment:
Remember we discussed this and I thought we settled on starting all JS data types with capital letters: Number, String.

Reply:
Right, sorry, fixed